webpack/assets/javascripts/react_app/components/AutoComplete/AutoCompleteActions.js

```diff
@@ -121,9 +134,20 @@ export const initialUpdate = (searchQuery, controller) => dispatch =>
       trigger: TRIGGERS.COMPONENT_DID_MOUNT,
       status: STATUS.RESOLVED,
       results: [],
+      url,
+      isDisabled,
     },
   });
 
+export const updateDisability = isDisabled => (dispatch) => {
```

Comment:
dispatch is redundant here

Reply:
How would it work without dispatch than ?

Reply:
This line is the magic that allows us to forget about dispatching (`mapDispatchToProps`):
https://github.com/theforeman/foreman/pull/6142/files#diff-a26c441a961320c4a75bac2b875a6ac5R35

Reply:
That's awesome, I was new to that.. fixed in all relevant places :)